### DISCUSSION TITLE
Can I run Python scripts in DBeaver ?

### DISCUSSION THREAD

**Discussion by MjAd7** [dbeaver/dbeaver#20332] [Q&A]
I'm trying to use Dbeaver while running python scripts, is it possible? and how can I do it because couldn't find useful guidance on the iternet?

**Comment by E1izabeth** [comment]
It's not clear what are you asking for. Do you want to run python script IN DBeaver or run python scripts WHILE working in DBeaver?

**Reply by MjAd7** [reply]
Run python script in DBeaver. For example, I be able to run this line : "import psycopg2".

**Reply by mdrashedulcom** [reply]
I have the same question. Can We use DBeaver to write python code instead of pycharm?

**Reply by E1izabeth** [reply]
There is no such opportunity.
